<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSheetWriter</class>
 <widget class="QMainWindow" name="MusicSheetWriter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Music Sheet Writer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/assets/logo.png</normaloff>:/assets/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">/*background-color: rgb(63, 81, 181);*/</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="welcomeWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #3f51b5;</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="1">
            <widget class="QPushButton" name="welcomeCreateNewProject">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>73</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(74,78,208); border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
outline: none;
color: #ffffff;</string>
             </property>
             <property name="text">
              <string>Cr√©er un projet</string>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/images/Accueil/add_project.png</normaloff>:/images/Accueil/add_project.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="welcomeOpenProject">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>73</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(74,78,208); border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
outline: none;
color: #ffffff;</string>
             </property>
             <property name="text">
              <string>Ouvrir un projet </string>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/images/Accueil/documents-icon.png</normaloff>:/images/Accueil/documents-icon.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="welcomeImportProject">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>73</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(74,78,208); border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
outline: none;
color: #ffffff;</string>
             </property>
             <property name="text">
              <string>Importer un projet</string>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/images/Accueil/import-icon.png</normaloff>:/images/Accueil/import-icon.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="welcomeLogo">
             <property name="maximumSize">
              <size>
               <width>940</width>
               <height>150</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Cinzel</family>
               <pointsize>25</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <stylestrategy>PreferDefault</stylestrategy>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">image: url(:/images/Accueil/logo_white.png);
image-position: top;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignHCenter|Qt::AlignTop</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <property name="indent">
              <number>-1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="1">
            <widget class="QWidget" name="mainWidgets" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>1140</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <property name="spacing">
               <number>9</number>
              </property>
              <item row="0" column="0">
               <widget class="QWidget" name="topWidgets" native="true">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="sizeConstraint">
                  <enum>QLayout::SetNoConstraint</enum>
                 </property>
                 <property name="leftMargin">
                  <number>5</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>5</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="generatorView" native="true">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="playerView" native="true">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="metronomeView" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0" rowspan="4">
               <widget class="QWidget" name="centerWidgets" native="true">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(240, 240, 240);</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="2">
                  <widget class="QWidget" name="scoreView" native="true"/>
                 </item>
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="1" alignment="Qt::AlignVCenter">
                  <widget class="QToolButton" name="btnMultipageLeft">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3" alignment="Qt::AlignVCenter">
                  <widget class="QToolButton" name="btnMultipageRight">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1" rowspan="5">
               <widget class="QWidget" name="editorView" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>220</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: white;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QWidget" name="projectsWidget" native="true">
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QTreeWidget" name="treeWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>120</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::WheelFocus</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">QTreeWidget {
background-color: rgb(48, 63, 159);
        border-style: none;
        border: none;
}

QTreeWidget::item {
        border: 2px solid transparent;
        color: white;
}

QTreeWidget::item:selected {
        border-left-color: #4caf50;
}

QTreeWidget::item:!selected {
        border-left-color: white;
}

QHeaderView::section {
        height: 43px;
        background-color: rgb(48, 63, 159);
        border: none;
        border-bottom: 1px solid #b6b6b6;
        font: 57 17pt &quot;Roboto Medium&quot;;
        color: white;
        padding-left: 10px;
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="midLineWidth">
                 <number>0</number>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="indentation">
                 <number>5</number>
                </property>
                <property name="rootIsDecorated">
                 <bool>false</bool>
                </property>
                <property name="itemsExpandable">
                 <bool>false</bool>
                </property>
                <property name="expandsOnDoubleClick">
                 <bool>false</bool>
                </property>
                <attribute name="headerVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="headerMinimumSectionSize">
                 <number>50</number>
                </attribute>
                <column>
                 <property name="text">
                  <string notr="true">Projets</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
